src/core/public/index.scss

```
@@ -7,5 +7,54 @@
 // Mixins provide generic code expansion through helpers
 @import '@elastic/eui/src/global_styling/mixins/index';
 
+$kbnGlobalNavClosedWidth: 53px;
+$kbnGlobalNavOpenWidth: 180px;
+$kbnGlobalNavLogoHeight: 70px;
+$kbnGlobalNavAppIconHeight: $euiSizeXXL + $euiSizeXS;
+
+#kibana-body {
+  overflow-x: hidden;
+  min-height: 100%;
+}
+
+.app-wrapper {
+  display: flex;
+  flex-flow: column nowrap;
+  position: absolute;
+  left: $kbnGlobalNavClosedWidth;
```

Well there was this one usage but it was both incorrect and overridden with the correct value by the Header component.
```suggestion
  left: 0;
```